### PR TITLE
Labb 9

### PR DESCRIPTION
**Pull request by DenisMilkevich**

### CODE REVIEW

File: lecture_notes/09 functions/3 - default.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def max(a=0, b=0):
+    if a > b:
+        return a
+    else:
+        return b
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/3 - default.py:5]: Use ternary operator and/or lambda syntax

File: lecture_notes/09 functions/2 - scope.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+print(['London', 'Beijing', 'Kiev'])
+print(to_upper_case(['London', 'Beijing', 'Kiev']))
+
+
+def count_len(cities):
+    return [len(city) for city in cities]
+
+
+print(cities)
+print(count_len(cities))
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/2 - scope.py:32]: Use def to create nested functions with nested contexts and variable name overlapping

**Review by tshemsedinov — CHANGES_REQUESTED** [review]

File: lecture_notes/09 functions/3 - default.py (hunk 1 above)
Comment (DenisMilkevich) [reply lecture_notes/09 functions/3 - default.py]: Done!

File: lecture_notes/09 functions/5 - procedure.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+def add_function(a, b):
+    result = a + b
+    return result
+
+
+print([add_function(10, 20), add_function(1, 2),
+       add_function(100, 20), add_function(100, 200)])
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/5 - procedure.py:7]: multiple lines will be more readable

File: lecture_notes/09 functions/3 - default.cpp [hunk 4]
```diff
@@ -0,0 +1,12 @@
+using namespace std;
+#include <iostream>https://github.com/DenisMilkevich/introduction-to-it/tree/Labb_9
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/3 - default.cpp:2]:
```suggestion
#include <iostream>
```

**Comment by tshemsedinov** [comment]
:+1:

**Pull request closed without merging**